bot/cogs/filtering.py

```diff
@@ -369,6 +398,48 @@ def mod_log(self) -> ModLog:
         except discord.errors.Forbidden:
             await channel.send(f"{filtered_member.mention} {reason}")
 
+    async def _scheduled_task(self, msg: dict) -> None:
```

Comment:
This function name does not seem descriptive enough.

Reply:
It's because it's an override/implementation of the abstract method defined in the Scheduler base class:

https://github.com/python-discord/bot/blob/7dc7ca3427cdd4b2fd1b1083d368e80f55207d9a/bot/utils/scheduling.py#L22-L32

Reply:
Oh, all right. Let's resolve this then.